src/index.html

```diff
@@ -6,7 +6,400 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="container">
+    <header class="header container__header">
```

Comment:
I think header isn't an element of container, this is separate blok, so, maybe, container__header isn't necessary

Reply:
yep! thanks!